### PR TITLE
DW-4595 improve k2hb handling

### PR DESCRIPTION
**Pull request by mark-r-m**
Improve k2hb handling. 
Add load test target for local testing. 
Default the container version for local dev. 
Make client id volatile, so every launch of one makes a new client in the Kafka Broker Consumer Group.
Split HBase error types into Connection and Write.

### CODE REVIEW

File: src/main/kotlin/Shovel.kt [hunk 1]
```diff
@@ -48,13 +49,17 @@ fun shovelAsync(consumer: KafkaConsumer<ByteArray, ByteArray>, hbase: HbaseClien
...
                     printLogs(consumer, offsets, usedPartitions)
                 }
 
-            } catch (e: HbaseReadException) {
+            } catch (e: HbaseConnectionException) {
+                logger.error("Error connecting to Hbase", e)
+                cancel(CancellationException("Error connecting to Hbase ${e.message}", e))
+            } catch (e: HbaseWriteException) {
                 logger.error("Error writing to Hbase", e)
                 cancel(CancellationException("Error writing to Hbase ${e.message}", e))
```
Comment (steveburton4) [inline src/main/kotlin/Shovel.kt:64]: Should we still be cancelling here? Wont this crash the jar still?

File: src/main/kotlin/Shovel.kt [hunk 2]
```diff
@@ -24,14 +25,14 @@ fun shovelAsync(consumer: KafkaConsumer<ByteArray, ByteArray>, hbase: HbaseClien
...
 
-
                 logger.info(
                     "Polling",
                     "poll_timeout", pollTimeout.toString(),
                     "topic_regex", Config.Kafka.topicRegex.pattern()
                 )
 
                 val records = consumer.poll(pollTimeout)
+                //consumer.pause(consumer.assignment())
```
Comment (steveburton4) [inline src/main/kotlin/Shovel.kt:35]: What is the plan with this and the resume? We should uncomment or take out really.

File: src/main/kotlin/Shovel.kt (hunk 1 above)
Comment (mark-r-m) [reply src/main/kotlin/Shovel.kt:64]: yes but we are going with the timeout and scale and changes as discussed :-)

File: src/main/kotlin/Shovel.kt (hunk 2 above)
Comment (mark-r-m) [reply src/main/kotlin/Shovel.kt:35]: oops yeah

**Pull request merged**